tracer/test/benchmarks/Benchmarks.Trace/TraceProcessorBenchmark.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Linq;
+using BenchmarkDotNet.Attributes;
+using Datadog.Trace;
+using Datadog.Trace.Processors;
+
+namespace Benchmarks.Trace
+{
+    [MemoryDiagnoser]
+    [BenchmarkAgent5]
+    public class TraceProcessorBenchmark
```

Comment:
Do we definitely want to run these on every commit? 🤔 These are currently used for only CI app and Stats aggregation right?

Reply:
You're right. I disabled it in CI, I want to run it only in local for now.